core/types/celo_additions.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+func (h *Header) IstanbulExtra() (*IstanbulExtra, error) {
```

Comment:
```suggestion
// IstanbulExtra returns the 'Extra' field of the header deserialized into an
// IstanbulExtra struct, if there is an error deserializing the 'Extra' field
// it will be returned.
func (h *Header) IstanbulExtra() (*IstanbulExtra, error) {
```